src/toil_scripts/lib/urls.py

```diff
@@ -8,7 +8,7 @@
 from toil_scripts.lib.programs import docker_call
 
 
-def download_url(url, work_dir='.', name=None, s3_key_path=None, cghub_key_path=None):
+def download_url(url, work_dir='.', name=None, synapse_login=None, s3_key_path=None, cghub_key_path=None):
```

Comment:
I thought we discussed at scrum that we weren't supporting synapse? It requires a user to hardcode a password into a plain text file.